kview/src/main/java/dev/ikm/komet/kview/controls/KLComponentSetControl.java

```diff
@@ -60,11 +60,11 @@ public final void setTitle(String value) {
     /**
      * This property holds the list of {@link Entity Entities} that have been added to the control
      */
-    private final ListProperty<EntityProxy> entitiesProperty = new SimpleListProperty<>(FXCollections.observableList(new LinkedList<>()));
-    public final ListProperty<EntityProxy> entitiesProperty() {
+    private final SetProperty<EntityProxy> entitiesProperty = new SimpleSetProperty<>(FXCollections.observableSet(new HashSet<>()));
```

Comment:
Can we change `SetProperty<EntityProxy>` to `ObjectProperty<IntIdSet>`?
Make use of the `IntIds.set` objects to create. 
If we need `entitiesProperty` property variable to get things to work temporarily we should at least create a `valueProperty()` of type `ObjectProperty<IntIdSet>`.